include/convex_bodies/ellipsoid.h

```diff
@@ -97,9 +112,20 @@ typedef Point PointType;
     }
 
 
-    void print() {
+    MT Lcov() const {
+        return _L_cov;
+    }
+
+
+    // return L_cov * x
+    VT mult_Lcov(VT const& x) const {
+        return _L_cov.template triangularView<Eigen::Lower>() * x;
+    }
+
+
+    void print() const {
         std::cout << "Ellipse is in the form: x' A x <= 1, (center is assumed to be origin always) \n";
-        std::cout << "c = \n" << c.print();
+        std::cout << "c = \n"; c.print();
```

Comment:
could you separate in two lines?

Reply:
removed this line, as `c` variable is removed now (as center is always origin)